publish-subscribe/subscriber.bal

```diff
@@ -0,0 +1,40 @@
+import ballerinax/kafka;
+import ballerina/websocket;
+
+service /ws on new websocket:Listener(8081) {
+    private final kafka:Consumer kafkaConsumer;
+    private final map<()> onGoingMatches = {};
+
+    function init() returns error? {
+        self.kafkaConsumer = check new (kafka:DEFAULT_URL, {
+            groupId: "unique-group-id",
+            topics: self.onGoingMatches.keys()
+        });
+    }
+
+    isolated resource function get [string matchId]() returns websocket:Service|error {
+        if !self.onGoingMatches.hasKey(matchId) {
+            self.onGoingMatches[matchId] = ();
+            check self.kafkaConsumer->subscribe(self.onGoingMatches.keys());
```

Comment:
If a new new user trys to watch match A now the old users who was listing to match B will also get match A updates. Looks like a bug.

Reply:
Yes. There is a bug. I'll fix this.

Reply:
Fixed.